packages/app/src/app/pages/common/Modals/ForkFrozenSandboxModal/ForkFrozenSandboxModal.tsx

```diff
@@ -47,8 +47,10 @@ export const ForkFrozenSandboxModal = observer(() => {
             modalClosed();
           }}
         >
-          Unlock
-          <Unlock />
+          <>
+            Unlock
+            <Unlock />
+          </>
```

Comment:
Hmm, I guess that file didn't get added to my commit.

What we really need to do here is change the `children` prop on the Button element to be `React.ReactNode` instead of `React.ReactElement | string`. I think it didn't get included because I edited the types in `lib` instead of `src`. I'll push a fix for this.

Reply:
`children` should indeed always be `React.ReactNode`, except for very rare situations. 👊